src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
+        ["bg"] = new("Bulgarian", "Български", "bg", "bul"),
+        ["bho"] = new("Bhojpuri", "भोजपुरी", "bho", "bho", TranslationServices.Google),
+        ["bm"] = new("Bambara", "ߓߊߡߊߣߊ߲ߞߊ߲", "bm", "bam", TranslationServices.Google),
         ["bn"] = new("Bengali", "বাংলা", "bn", "ben"),
+        ["bo"] = new("Tibetan", "བོད་སྐད་", "bo", "bod", TranslationServices.Bing | TranslationServices.Microsoft),
         ["bs"] = new("Bosnian", "bosanski", "bs", "bos"),
-        ["bg"] = new("Bulgarian", "Български", "bg", "bul"),
-        ["my"] = new("Burmese", "မြန်မာ", "my", "mya"),
         ["ca"] = new("Catalan", "Català", "ca", "cat"),
         ["ceb"] = new("Cebuano", "Binisaya", "ceb", "ceb", TranslationServices.Google | TranslationServices.Yandex),
-        ["ny"] = new("Chichewa", "Nyanja", "ny", "nya", TranslationServices.Google),
-        ["zh-CN"] = new("Chinese (Simplified)", "中文 (简体)", "zh-CN", "zho-CN"),
-        ["zh-TW"] = new("Chinese (Traditional)", "繁體中文 (繁體)", "zh-TW", "zho-TW", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
         ["co"] = new("Corsican", "Corsu", "co", "cos", TranslationServices.Google),
-        ["hr"] = new("Croatian", "Hrvatski", "hr", "hrv"),
         ["cs"] = new("Czech", "Čeština", "cs", "ces"),
+        ["cv"] = new("Chuvash", "Чӑвашла", "cv", "chv", TranslationServices.Yandex),
+        ["cy"] = new("Welsh", "Cymraeg", "cy", "cym"),
         ["da"] = new("Danish", "Dansk", "da", "dan"),
-        ["nl"] = new("Dutch", "Nederlands", "nl", "nld"),
+        ["de"] = new("German", "Deutsch", "de", "deu"),
+        ["dgo"] = new("Dogri", "डोगरी (ḍogrī) ڈوگرى", "dgo", "dgo", TranslationServices.Google),
+        ["dv"] = new("Divehi", "ދިވެހިބަސް", "dv", "div", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["ee"] = new("Ewe", "Eʋegbe", "ee", "ewe", TranslationServices.Google),
+        ["el"] = new("Greek", "Ελληνικά", "el", "ell"),
+        ["emj"] = new("Emoji", "Emoji", "emj", "emj", TranslationServices.Yandex), // Not present in Yandex.Cloud
         ["en"] = new("English", "English", "en", "eng"),
+        ["en-GB"] = new("English (UK)", "English (UK)", "en-GB", "en-GB", TranslationServices.Google),
         ["eo"] = new("Esperanto", "Esperanto", "eo", "epo", TranslationServices.Google | TranslationServices.Yandex),
+        ["es"] = new("Spanish", "Español", "es", "spa"),
+        ["es-MX"] = new("Spanish (Mexico)", "español (México)", "es-MX", "es-MX", TranslationServices.Bing | TranslationServices.Microsoft),
         ["et"] = new("Estonian", "Eesti", "et", "est"),
+        ["eu"] = new("Basque", "Euskara", "eu", "eus"),
+        ["fa"] = new("Persian", "فارسی", "fa", "fas"),
         ["fi"] = new("Finnish", "Suomi", "fi", "fin"),
+        ["fil"] = new("Filipino", "Tagalog", "fil", "fil", TranslationServices.Bing | TranslationServices.Microsoft),
```

Comment:
Isn't the native name "Filipino"?

Reply:
No.

Reply:
Did you read the link I posted?

https://learningfilipino.com/blog/difference-between-tagalog-and-filipino/

Although Tagalog and Filipino are technically supposed to be separate languages, many (including Pilipino's) see them as one and the same.

Alternatively, **Pilipino** could be used as the native name.
> On the other hand, Pilipino, is how the locals from the Philippines refer to themselves, or to their national language.

Above quote comes from the following link:
http://www.hawaii.edu/cps/filipino.html

Reply:
I know, but I was referring to how Filipinos refer to its name in their language. From Wikipedia's page it's referred as just "Filipino" or "Wikang Filipino".

Reply:
I've changed it to Pilipino.

Reply:
> I know, but I was referring to how Filipinos refer to its name in their language. From Wikipedia's page it's referred as just "Filipino" or "Wikang Filipino".

In the wiki, do keyword search for "Pilipino".
You'll find it in Further history.  The reason why the native name should not be Filipino is because most Pilipino languages do not have the "F" sound.

Filipino is the anglicize version of the name.

Reply:
From Wikipedia:

> A compromise was reached and the wording on the 1973 constitution made no mention of dropping the national language Pilipino or made any mention of Tagalog. Instead, the 1973 Constitution, in both its original form and as amended in 1976, designated English and Pilipino as official languages and provided for development and formal adoption of a common national language, termed Filipino, to replace Pilipino.

From https://learningfilipino.com/blog/difference-between-tagalog-and-filipino/:

> You see Tagalog was one of the native languages of the Philippines. It was a native language long before the Spaniards and Americans colonized the Philippines. At some point, Filipinos wanted a native language to become the national language of the Philippines.  And Tagalog was the chosen native language to become the basis for this national language. Tagalog was then renamed “Pilipino” and eventually “Filipino.”

> Renaming of Pilipino to Filipino

> By 1987, “Pilipino” was renamed “Filipino”, the one letter change from “p” to “f” marking a significant shift in the Filipino language. The 20-letter Tagalog abakada was replaced with the 28-letter [Filipino alphabet](https://learningfilipino.com/blog/language/filipino-alphabet/), and included letters from the English and Spanish alphabet not present in the former ABaKaDa alphabet.


It seems that at some point Filipino was named Pilipino but in 1987 it was changed to Filipino.


Even in [Wikipedia in Tagalog](https://tl.wikipedia.org/wiki/Wikang_Filipino) Filipino is referred as just "Filipino".

I took the native names of the languages from Wikipedia and from Microsoft's own API endpoint for languages (https://api.cognitive.microsofttranslator.com/languages?api-version=3.0&scope=translation).
In both places the native name of Filipino is "Filipino" so I don't think Wikipedia and Microsoft would be incorrect about their language data.
![image](https://user-images.githubusercontent.com/39978852/188395278-4a0bd23b-65dd-459f-93db-9e75b51f0efa.png)

Reply:
> I took the native names of the languages from Wikipedia and from Microsoft's own API

Considering how they're handling "pt" as PT-Brazil, I wouldn't consider them an expert on the matter.

The proceeding quote comes from the following article: 
[https://learningtagalog.com/articles/tagalog_filipino_pilipino_difference.html](https://learningtagalog.com/articles/tagalog_filipino_pilipino_difference.html)
> "Filipino is still the same as Tagalog, which unless otherwise specified, is generally understood as standard or Manila Tagalog. The grammar is identical."


In any case, I give up. I'll change it to Filipino until after the update.

Reply:
> Considering how they're handling "pt" as PT-Brazil, I wouldn't consider them an expert on the matter.

Every translator treats it like that, and that's because Brazilian portuguese is the most common variant of the two.